handlerWind.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+)
+
+func handlerGetWind(w http.ResponseWriter, r *http.Request) {
+	// Extracting the latitude and longitude from the Query
+	log.Println(r.URL.Query())
+	queryStr := r.URL.Query()
```

Comment:
The following code, including the call to write json file, are repeated for both `basic` and `wind`  handler functions. I guess that it may be repeated for more handlers later. Would it make sense to bunch them into a separate function block and call then call it?

Reply:
So like only 1 function for both?

Reply:
No, I mean the checks that are necessarily repeated for both handlers can be moved to another function. For example, the lat long coordinates are mandatory for both handler functions and they are required for other apis as well. That can be moved out, along with request timeouts and such things that are commonly repeated.

Reply:
I see. But in hindsight, i technically repeat the code for both handlers anyway so can i not just make it as one function?

Reply:
Or is it better to have 1 handler for each route?